src/components/Participant/ParticipantCard.js

```diff
@@ -3,28 +3,22 @@ import styled from 'styled-components';
 export default function ParticipantCard({name, businessType, address, liveDate, precinct}) {
   return (
     <ParticipantContainer role="article">
-      <span>{name}</span>
-      <span>Business Type: {businessType}</span>
-      <span>Address: {address}</span>
-      <span>Since: {liveDate}</span>
-      <span>Precint: {precinct}</span>
+      <h3>{name}</h3>
+      <p>Business Type: {businessType}</p>
+      <p>Address: {address}</p>
+      <p>Since: {liveDate}</p>
+      <p>Precint: {precinct}</p>
```

Comment:
As far as I can see you have an `h1` for your app name but no `h2` element. So this `h3` should be an `h2`. If it is too big, you can style it as you wish. As well you could think about using an`ul` here instead of the `p`-tags.

Reply:
unfortunately when i use `ul` this error message appears in the console: `Uncaught TypeError: styled_components__WEBPACK_IMPORTED_MODULE_1__.default.list is not a function`